pypeit/core/wavecal/templates.py

```diff
@@ -764,6 +764,13 @@ def main(flg):
         slits = [1026, 1021]
         lcut = [4350.0, 8000.0]
         build_template([wfile1, wfile2], slits, lcut, binspec, outroot, lowredux=False, normalize=True)
+        # FeAr BL
```

Comment:
this is ok, but I'm trying to move all of these into separate modules
in the `spectrographs` folder.

Reply:
Sorry about this... If I should be doing something else, just let me know (I've written the docs such that this method is valid - sorry if I'm out of date!).

Reply:
this is fine.  But, yeah, see this an example for 
the future:
https://github.com/pypeit/PypeIt/blob/release/pypeit/core/wavecal/spectrographs/templ_keck_deimos.py